survey/HTML-syntax-in-EPUB.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Survey on the Impact of Allowing HTML Syntax in EPUB
+date: 2025-05-08
+---
+
+The Publishing Maintenance Working Group (PMWG) is exploring the possibility of allowing HTML as an alternative content document format in EPUB 3.4, alongside the existing XHTML format.
+
+This initiative aims to address the diminishing use and maintenance of XHTML outside the publishing industry while leveraging HTML's status as a living standard. 
```

Comment:
```suggestion
This initiative aims to address the diminishing use and maintenance of the XML syntax outside the publishing industry while leveraging the future evolution of HTML. 
```

The "Living standard" terminology is not necessarily known outside the standard world, and may be frightening for outsiders...